server/simple_server.go

```diff
@@ -98,6 +99,17 @@ func (s *SimpleServer) safelyExecuteRequest(w http.ResponseWriter, r *http.Reque
 		}
 	}()
 
+	// lookup metric name if we can
+	metricName := r.URL.Path
```

Comment:
nit: it's not really the name of the metric but the originally registered path of the endpoint. maybe `registeredPath` ?

Reply:
done

Reply:
Looks good!